examples/rossmann-store-sales-example.ipynb

```diff
@@ -192,20 +185,20 @@
     "### Datasets\n",
     "In general, the `Op`s in our `Workflow` will require measurements of statistical properties of our data in order to be leveraged. For example, the `Normalize` op requires measurements of the dataset mean and standard deviation, and the `Categorify` op requires an accounting of all the categories a particular feature can manifest. However, we frequently need to measure these properties across datasets which are too large to fit into GPU memory (or CPU memory for that matter) at once.\n",
     "\n",
-    "NVTabular solves this by providing the `dataset` object, an iterator over manageable chunks of sets of parquet or csv files that can we can use to compute statistics in an online fashion (and, later, to train neural networks in batches loaded from disk). The size of those chunks will be determined by the `gpu_memory_frac` kwarg, which will load chunks whose memory footprint is equal to that fraction of available GPU memory.\n",
+    "NVTabular solves this by providing the `Dataset` class, which breaks a set of parquet or csv files into into a collection of `cudf.DataFrame` chunks that can fit in device memory.  Under the hood, the data decomposition corresponds to the construction of a [`dask_cudf.DataFrame`](https://docs.rapids.ai/api/cudf/stable/dask-cudf.html) object.  By representing our dataset as a lazily-evaluated [Dask](https://dask.org/) collection, we can handle the calculation of complex global statistics (and later, can also iterate over the partitions while feeding data into a neural network). The size of the underlying `dask_cudf.DataFrame` partitions (chunks) is determined by the `part_mem_fraction` kwarg, which corresponds a desired fraction of total GPU memory.\n",
     "\n",
     "Larger chunks will lead to shorter run times due to the parallel-processing power of GPUs, but will constrain your memory and possibly lead to disk caching by expensive operations, thereby lowering efficiency."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [],
    "source": [
     "GPU_MEMORY_FRAC = 0.2\n",
-    "train_dataset = nvt.Dataset(TRAIN_PATH, gpu_memory_frac=GPU_MEMORY_FRAC, columns=COLUMNS)\n",
-    "valid_dataset = nvt.Dataset(VALID_PATH, gpu_memory_frac=GPU_MEMORY_FRAC, columns=COLUMNS)"
+    "train_dataset = nvt.Dataset(TRAIN_PATH, part_mem_fraction=GPU_MEMORY_FRAC)\n",
+    "valid_dataset = nvt.Dataset(VALID_PATH, part_mem_fraction=GPU_MEMORY_FRAC)"
```

Comment:
Sounds good - Thanks Ben!

There are certainly good reasons to avoid using kwargs.  In this case, given that I don't trust the stability of the downstream API, I'd prefer something like `csv_options`/`dask_reader_options` over a list of many pass-through arguments.

In general - Feel free to complain whenever you see `kwargs` code from me, and I'll be happy to make things more explicit :)